bundles/edu.kit.kastel.sdq.eclipse.grading.client/src/edu/kit/kastel/sdq/eclipse/grading/client/rest/ArtemisStudentClient.java

```diff
@@ -0,0 +1,9 @@
+package edu.kit.kastel.sdq.eclipse.grading.client.rest;
+
+public class ArtemisStudentClient extends ArtemisClient {
+	
+	public ArtemisStudentClient(final String username, final String pwd, final String url) {
+		super(username, pwd, url);
+	}
+
+}
```

Comment:
Why this class? Do you want to put methods for students here? If so, that's fine :)